docs/attributes-registry/authentication.md

```diff
@@ -0,0 +1,15 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Authentication
+
+## Authentication Attributes
```

Comment:
I think this should be inside `enduser.md`. If you add it like this, it will introduce a new "area" called `authentication` which is not what happens in reality. The authentication attributes are inside `enduser` which is the area. Please move it to `enduser.md`